eli5/_shap_explanation_summary.py

```diff
@@ -0,0 +1,81 @@
+from eli5._feature_weights import get_top_features_filtered
+from eli5.base import Explanation, TargetExplanation
+from eli5._feature_importances import get_feature_importance_explanation
+
+DECISION_PATHS_DESCRIPTION = """
+Feature weights are calculated by following decision paths in trees
```

Comment:
I think all this unused descriptions and code is because you copy&paste the code from elsewhere and then you worked with it. Could you please remove this code when commiting? It would simplify my review labour :-) In general try to have your code as clean as possible also during development, as it is a good practice.

Reply:
Yes! will do so. I was trying to mimic a module.